test/foundry/unit/StakingRewardsV2/StakingRewardsV2.t.sol

```diff
@@ -65,21 +65,9 @@ contract StakingRewardsV2Test is DefaultStakingV2Setup {
         stakingRewardsV2.recoverERC20(address(kwenta), 0);
     }
 
-    function test_Only_RewardEscrowCan_Call_unstakeEscrowSkipCooldown() public {
```

Comment:
**Issue (blocking):** There should be a test of this kind for `unstakeEscrowAdmin`.

Reply:
agreed ^

Reply:
Added a new test for this : `test_Only_RewardEscrow_Can_Call_unstakeEscrowAdmin`